Trappist/src/Promact.Trappist.Repository/TestConduct/TestConductRepository.cs

```diff
@@ -295,7 +295,7 @@ public TestConductRepository(TrappistDbContext dbContext, ITestsRepository testR
                 code.Input = testCase.TestCaseInput;
                 var result = await ExecuteCodeAsync(code);
 
-                if(result.ExitCode == -34494 || result.CompilerOutput != "")//EXITCODE if code run fails
+                if (result.ExitCode == -34494 || result.CompilerOutput != "")//EXITCODE if code run fails
```

Comment:
you need to document error codes, otherwise simplify it, consumer has no way to know what 34494 means